app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -98,6 +133,10 @@ private fun ImportFilesScreen(
 ) {
     val context = LocalContext.current
     var showUploadSourceChoiceBottomSheet by rememberSaveable { mutableStateOf(initialShowUploadSourceChoiceBottomSheet) }
+    var showValidityPeriodBottomSheet by rememberSaveable { mutableStateOf(false) }
+    var showDownloadLimitBottomSheet by rememberSaveable { mutableStateOf(false) }
+    var showPasswordOptionBottomSheet by rememberSaveable { mutableStateOf(false) }
+    var showEmailLanguageBottomSheet by rememberSaveable { mutableStateOf(false) }
```

Comment:
If we had a single state that contains the enum of the currently opened bottom sheet or null otherwise, it could simplify some parts of the code. Have you considered this option?

Reply:
It seems it recomposes the all the options' BottomSheet Scaffold each times now, but this isn't a real performance issue in this case so I'll go with this way